environment.yaml

```diff
@@ -58,15 +59,19 @@ dependencies:
   - r-circular
   - r-cluster
   - r-ccremover
+  - r-conicfit
+  - r-corshrink
   - r-cowplot
   - r-data.table
   - r-dplyr
   - r-dt
   - r-gdata
+  - r-genlasso
   - r-ggplot2
   - r-gplots
   - r-heatmap3
   - r-mclust
+  - r-parallel
```

Comment:
Could you please remove the entry `r-parallel`? `r-parallel` is included as part of `r-base`, so it can't be installed separately.

```
$ conda install r-parallel
Fetching package metadata ...............

PackageNotFoundError: Packages missing in current channels:
            
  - r-parallel

```